### PR TITLE
Ruby Bubble Sort Project

### PR DESCRIPTION
**Pull request by nusretbutunay**
Advanced Building Blocks - Bubble Sort

[Collaborative project]
In this project, we implemented the bubble sort algorithm. The assignment is focused on the use of yield and passing a block to a method.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        temp = arr[i]
+        arr[i] = arr[i + 1]
+        arr[i + 1] = temp
+        swap = true
+      end
+      i += 1
+    end
+    i = 0
+  end
+  puts arr
```
Comment (mosesogwo) [inline app.rb]: You should return arr, not puts

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      end
+      index += 1
+    end
+    index = 0
+  end
+  numbers
+end
+
+numbers = [4, 3, 78, 2, 0, 2]
+puts bubble_sort(numbers)
```
Comment (mosesogwo) [inline app.rb]: Let's remove this from the file

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Great job!:+1:, I like your PR title. You should however fix the following:**
:point_right: bubble_sort_by should return the sorted array and not print it.
:point_right: remove debugging code from your file

_**Kindly make those changes and submit another code review request.**_

**Any questions? You can reach me on Slack (Moses Ogwo)**

**Cheers:stars:**

### Status: Changes Requested.

**Comment by Dandush03** [comment]
# Status: Project Approved

## Good luck in your next assignment 

# Happy Coding 💻

**Pull request merged**